RecoMET/METAlgorithms/src/METSignificance.cc

```diff
@@ -39,6 +39,7 @@ metsig::METSignificance::METSignificance(const edm::ParameterSet& iConfig) {
   jetEtas_ = cfgParams.getParameter<std::vector<double> >("jeta");
   jetParams_ = cfgParams.getParameter<std::vector<double> >("jpar");
   pjetParams_ = cfgParams.getParameter<std::vector<double> >("pjpar");
+  useDeltaRforFootprint_ = cfgParams.exists("useDeltaRforFootprint") ? cfgParams.getParameter<bool>("useDeltaRforFootprint") : false;
```

Comment:
fillPSetDescription is in order, in line with https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#A_Plugin_Module_Using_Another_He

Reply:
here as well, it looks like the useDeltaRforFootprint can just be added as required (in case implementation of the PSet validation is less motivating)